File: parser/props/DNASuitEdge.py
class DNASuitEdge:
    PROP_CODE = 22

    def __init__(self, startPoint, endPoint, zoneId):
        self.startPoint = startPoint
        self.endPoint = endPoint
        self.zoneId = zoneId

    def setStartPoint(self, startPoint):
        self.startPoint = startPoint

    def getStartPoint(self):
        return self.startPoint

    def setEndPoint(self, endPoint):
        self.endPoint = endPoint

    def getEndPoint(self):
        return self.endPoint

    def setZoneId(self, zoneId):
        self.zoneId = zoneId

    def getZoneId(self):
        return self.zoneId

File: src/suit/DNASuitEdge.cxx
#include "DNASuitEdge.h"

TypeHandle DNASuitEdge::_type_handle;

DNASuitEdge::DNASuitEdge(PT(DNASuitPoint) start_point, PT(DNASuitPoint) end_point,
                         zone_id_t zone_id): m_start_point(start_point),
                                             m_end_point(end_point),
                                             m_zone_id(zone_id)
{
}

DNASuitEdge::~DNASuitEdge()
{
}
